packages/stokhos/test/UnitTest/Stokhos_TpetraCrsMatrixUQPCEUnitTest.hpp

```diff
@@ -1436,6 +1426,7 @@ TEUCHOS_UNIT_TEST_TEMPLATE_4_DECL(
     MueLu::CreateTpetraPreconditioner<BaseScalar,LocalOrdinal,GlobalOrdinal,Node>(mean_matrix_op, *muelu_params);
   RCP<OP> M = rcp(new Stokhos::MeanBasedTpetraOperator<Scalar,LocalOrdinal,GlobalOrdinal,Node>(M_s));
 #else
+  typedef typename Storage::execution_space Device;
```

Comment:
I would prefer reserving the name "Device" for a specialization of `Kokkos::Device`.

Reply:
I added another commit to address this.